t/032_cancel_consume_ack_exception.t

```diff
@@ -0,0 +1,40 @@
+use Test::More tests => 17;
+use strict;
+use warnings;
+
+use FindBin qw/$Bin/;
+use lib "$Bin/lib";
+use NAR::Helper;
+
+my $helper = NAR::Helper->new;
+
+ok $helper->connect, "connected";
```

Comment:
Is it just me, or is nothing putting our channel into consume mode? It could totally be in the `NAR::Helper` somewhere, but I don't see it.